crates/ide-assists/src/handlers/auto_import.rs

```diff
@@ -156,6 +156,8 @@ pub(super) fn find_importable_node(
     {
         ImportAssets::for_method_call(&method_under_caret, &ctx.sema)
             .zip(Some(method_under_caret.syntax().clone().into()))
+    } else if let Some(_) = ctx.find_node_at_offset_with_descend::<ast::Param>() {
```

Comment:
I'm not sure if `if let Some(_) = ...` rather than `.is_some()` is the preferred style here but as every other if statement uses `let Some(...) = ...` I thought I'd do the same